src/app/extensions/quoting/store/quote-request/quote-request.effects.ts

```diff
@@ -366,6 +366,17 @@ export class QuoteRequestEffects {
     map(([quoteId]) => new actions.LoadQuoteRequestItems({ id: quoteId }))
   );
 
+  /**
+   * Triggers a LoadQuoteRequestItems action if a quote request gets selected and LoadQuoteRequestsSuccess action triggered
```

Comment:
either no comment (I think this effect is self-explanatory) or the right comment, please :sweat_smile: